rad/src/filters/deviation.rs

```diff
@@ -0,0 +1,431 @@
+use crate::error::RadError;
+use crate::filters::RadonFilters;
+use crate::operators::array::transpose;
+use crate::reducers;
+use crate::types::array::RadonArray;
+use crate::types::boolean::RadonBoolean;
+use crate::types::RadonType;
+use crate::types::RadonTypes;
+use serde_cbor::Value;
+
+pub fn standard(input: &RadonArray, extra_args: &[Value]) -> Result<RadonTypes, RadError> {
```

Comment:
```suggestion
pub fn standard_filter(input: &RadonArray, extra_args: &[Value]) -> Result<RadonTypes, RadError> {
```

could be confusing use the same method name for the filter and the reduce method